vrs/vrs_schema_renaming_script_temp.py

```diff
@@ -0,0 +1,72 @@
+"""
+ Script to update column names from GA4GH VRS-Python Branch 0.8.1 to 0.8.2 convention
+ See: https://github.com/ga4gh/vrs-python/releases/tag/0.8.2
+ Note that this script will be run on 05-04-23 or 05-05-23 as a temporary fix.
+These changes will be permenantly added to the script which adds the VRS Annotations in a later update.
+ Note that this code may not be fully productionized or up-to-date with the rest of the team's code standards. 
+"""
+
+import argparse
+import hail as hl
+
+
+def main(args):
+
+    import_path = args.import_ht
+    ht = hl.read_table(import_path)
+
+    # Code which actually changes naming schema from VRS-Python Branch 0.8.1 to 0.8.2
+
+    ht = ht.annotate(
+        info=ht.info.annotate(
+            vrs=ht.info.vrs.rename(
+                {
+                    "VRS_Allele": "VRS_Allele_IDs",
+                    "VRS_Start": "VRS_Starts",
+                    "VRS_End": "VRS_Ends",
+                    "VRS_Alt": "VRS_States",
+                }
+            )
+        )
+    )
+
+    ht_reformat = ht.annotate(
+        info=ht.info.annotate(
+            vrs=ht.info.vrs.annotate(
+                VRS_Allele_IDs=ht.info.vrs.VRS_Allele_IDs.split(","),
+                VRS_Starts=ht.info.vrs.VRS_Starts.split(",").map(lambda x: hl.int(x)),
+                VRS_Ends=ht.info.vrs.VRS_Ends.split(",").map(lambda x: hl.int(x)),
+                VRS_States=ht.info.vrs.VRS_States.split(","),
+            )
+        )
+    )
+
+    ht_reformat.info.vrs.show()
+
+    # Code to add global annotation for which VRS Version (not GitHub Branch Number) is used
+
+    ht = ht.annotate_globals(VRS_Version="v1.3")
```

Comment:
just making this lowercase as i don't see anything else currently capitalized in the release globals, such as 
vep_version
dbsnp_version

Reply:
yeah that makes sense. on a slightly different topic , some other INFO fields are capitalized so we don't have to worry about 'VRS_Allele_IDs' using capital letters though, correct?